rac_aspace/decorators.py

```diff
@@ -0,0 +1,9 @@
+
+
+def check_dictionary(func):
+    def inner(*args, **kwargs):
+        if not isinstance(args, dict):
```

Comment:
```suggestion
        if not isinstance(args[0], dict):
```
`args` is a tuple containing all the positional arguments passed to the function, so we'll need to get the first one OR alternatively, check to make sure ALL of the args are dicts, which (I think) would look like:

```suggestion
        if not [isinstance(a, dict) for a in args]:
```